test/torch_np/numpy_tests/lib/test_shape_base_.py

```diff
@@ -94,6 +94,7 @@ def test_empty(self):
         actual = take_along_axis(a, ai, axis=1)
         assert_equal(actual.shape, ai.shape)
 
+    @pytest.mark.xfail(reason="https://github.com/pytorch/pytorch/pull/107875")
```

Comment:
@guilhermeleobas if this PR lands before https://github.com/pytorch/pytorch/pull/108880#pullrequestreview-1618672157 you'll need to remove this xfail